packages/connector-tencent-email/src/types.ts

```diff
@@ -0,0 +1,62 @@
+import { z } from 'zod';
+
+export const FromInfo = z.object({
+  fromAddress: z.string().optional(),
+  fromName: z.string().optional(),
+  replyAddress: z.string().optional(),
+});
+
+export const ValueNames = [
+  'code',
+  'toAddress',
+  'fromAddress',
+  'fromName',
+  'replayAddress',
```

Comment:
```suggestion
  'replyAddress',
```

Reply:
And also I think you may have to directly define it as a type, instead of tinkering the string array and Record?
```ts
export type EmailTemplateProps = {
  code: string;
  toAddress: string;
  fromAddress: string;
  fromName?: string;
  replyToAddress?: string;
  subject?: string;
};
```